Cabal/Distribution/PackageDescription/Check.hs

```diff
@@ -1432,9 +1438,12 @@ checkLocalPathsExist ops pkg = do
   let dirs = [ (dir, kind)
              | bi <- allBuildInfo pkg
              , (dir, kind) <-
-                  [ (dir, "extra-lib-dirs") | dir <- extraLibDirs bi ]
-               ++ [ (dir, "include-dirs")   | dir <- includeDirs  bi ]
-               ++ [ (dir, "hs-source-dirs") | dir <- hsSourceDirs bi ]
+                  [ (dir, "extra-lib-dirs")  | dir <- extraLibDirs  bi ]
+               ++ [ (dir, "include-dirs")    | dir <- includeDirs   bi ]
+               ++ [ (dir, "hs-source-dirs")  | dir <- hsSourceDirs  bi ]
+               ++ [ (dir, "c-sources")       | dir <- cSources      bi ]
```

Comment:
Addition of `c-sources` and `js-sources` here is wrong, this generates spurious warnings like

```
'c-sources: cbits/getnumcores.c' directory does not exist.
```